packages/back-end/src/controllers/experiments.ts

```diff
@@ -1544,15 +1570,16 @@ export async function postSnapshot(
     hasRegressionAdjustmentFeature &&
     (regressionAdjustmentEnabled !== undefined
       ? regressionAdjustmentEnabled
-      : org.settings?.regressionAdjustmentEnabled ?? false);
+      : orgSettings?.regressionAdjustmentEnabled ?? false);
```

Comment:
Same comment - we are "calculating" settings here which I'm not sure is the best place to do